src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -391,6 +391,100 @@ std::set<std::vector<element::Type>> jit_exp_emitter::get_supported_precisions(c
     return {{element::f32}};
 }
 
+/// GELU_TANH ///
+jit_gelu_tanh_emitter::jit_gelu_tanh_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                             dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                             const std::shared_ptr<ov::Node>& node)
+        : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+    prepare_table();
+    tanh_emitter = std::make_unique<jit_tanh_emitter>(h, host_isa, node);
+}
+
+jit_gelu_tanh_emitter::jit_gelu_tanh_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                             dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                             const ov::element::Type exec_prc) : jit_emitter(host, host_isa, exec_prc) {
+    prepare_table();
+    tanh_emitter = std::make_unique<jit_tanh_emitter>(h, host_isa, exec_prc);
+}
+
+size_t jit_gelu_tanh_emitter::get_inputs_count() const { return 1; }
+
+size_t jit_gelu_tanh_emitter::get_aux_vecs_count() const {
+    return tanh_emitter->get_aux_vecs_count() + 2;
```

Comment:
```suggestion
    return std::max(tanh_emitter->get_aux_vecs_count() + 2ull, 3ull);
```
I see you asserting aux_vec_idxs.size() to be at least 5 in jit_gelu_tanh_emitter::emit_impl. I suggest we put constraints here. As far as I see, vmm_aux3 can reuse vmm_aux0, and vmm_aux_src can reuse vmm_aux2. So I put it 3 here instead of 5. What do you think?

Reply:
done, thanks